azurerm/resource_arm_virtual_machine_scale_set_test.go

```diff
@@ -298,6 +298,41 @@ func TestAccAzureRMVirtualMachineScaleSet_planManagedDisk(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMVirtualMachineScaleSet_customImage(t *testing.T) {
+	ri := acctest.RandInt()
+	resourceGroup := fmt.Sprintf("acctestRG-%d", ri)
+	userName := "testadmin"
+	password := "Password1234!"
```

Comment:
I guess one question about these kinds of test credentials being in the clear. I understand that the resources are ephemeral and more than likely this is low risk, but is there any way that maybe we could randomize this? That way, if by any chance the test instances/VMSS ends up available on the internet anywhere, at least the credentials are't predictable. Thoughts?

Reply:
Sounds sane sane to me!